app/views/ExposureHistory/NextSteps.tsx

```diff
@@ -59,18 +57,14 @@ const NextSteps = (): JSX.Element => {
 
 const styles = StyleSheet.create({
   container: {
-    flex: 1,
```

Comment:
Why the switch to percents here?

Reply:
This was written before your pr that fixed the layout stuff with NavigationWrapper.  Flex is what we want now.